components/lib/dataprotect/src/main/java/mozilla/components/lib/dataprotect/SecureAbove22Preferences.kt

```diff
@@ -59,9 +59,12 @@ private interface KeyValuePreferences {
  *
  * @param context A [Context], used for accessing [SharedPreferences].
  * @param name A name for this storage, used for isolating different instances of [SecureAbove22Preferences].
+ * @param forceInsecure A flag indicating whether to force plaintext storage. If set to `true`,
+ * [InsecurePreferencesImpl21] will be used as a storage layer.
```

Comment:
expand the comment? "[...], otherwise a storage implementation will be decided based on Android API version, with a preference given to secure storage."